### DISCUSSION TITLE
Searchable LaTeX formulas (MathJax)

### DISCUSSION THREAD

**Discussion by christophhillisch** [squidfunk/mkdocs-material#4654] [Q&A] [answered]
Hello,

I would be interested in making LaTeX formulas (implemented via MathJax) searchable.

As an example: 
I would like `$\mathrm{pIC}_{50}$` to be found when I type pIC50 in the search bar. Is there maybe a way to add this text hidden and searchable?

**Comment by squidfunk** [comment, marked as answer]
This might be possible to achieve with a clever use of the [search separator/tokenizer](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#tokenizer-lookahead), but it's definitely tricky.

Note that this is currently only available in [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/).

**Reply by christophhillisch** [reply]
Thanks for the quick reply.

So something like adding \$\\mathrm{ to the separator could work?

Is there an easy way to get a notification once features are available to non-insiders?

**Reply by squidfunk** [reply]
You can follow me on [Twitter](https://twitter.com/squidfunk). Other than that, you can check back in a few weeks. But why not [sponsor](https://squidfunk.github.io/mkdocs-material/insiders/#how-to-become-a-sponsor) and help support the project? 😉 You could use the new features immediately.
